CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- On Windows, fix deadlock caused by pressing multiple mouse buttons down at once.
```

Comment:
As the issues only persists in the unreleased master there is no need for a changelog entry

Reply:
Ok, will remove.